### PR TITLE
Fix typo in readme

### PR DESCRIPTION
**Pull request by svenluijten**
"find" -> "found"

I am targetting this branch, because it's a simple typo, which is completely backwards compatible.

## Subject

Typo fix.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Check out the documentation on the [official website](https://sonata-project.org
 
 For general support and questions, please use [StackOverflow](http://stackoverflow.com/questions/tagged/sonata).
 
-If you think you find a bug or you have a feature idea to propose, feel free to open a issue
+If you think you found a bug or you have a feature idea to propose, feel free to open a issue
```
Comment (greg0ire) [inline README.md]: Can you also fix "a issue" => "an issue" while you're at it?

**Comment by jordisala1991** [comment]
This needs to be done on dev-kit

**Comment by svenluijten** [comment]
I can open up another PR against the `develop` branch, if that's what you mean @jordisala1991?

**Comment by jordisala1991** [comment]
I mean that there is some files that are shared across bundles, we use a tool called dev-kit: https://github.com/sonata-project/dev-kit

This change needs to be done in here: https://github.com/sonata-project/dev-kit/blob/master/project/README.md

Can you open the PR on that repository?

**Comment by svenluijten** [comment]
Gotcha, sure! Opened at sonata-project/dev-kit#241

**Comment by jordisala1991** [comment]
Thanks! Closing this one

**Pull request closed without merging**